README.md

```diff
@@ -1,27 +1,23 @@
-# Docker-Mapserver
+<!-- 
+SPDX-FileCopyrightText: German Aerospace Center (DLR) <cosmoscout@dlr.de>
+SPDX-License-Identifier: CC0-1.0
+ -->
+# Map-Server Container for CosmoScout VR
 
-A dockerized mapserver for CosmoScout [csp-lod bodies](https://github.com/cosmoscout/cosmoscout-vr/tree/main/plugins/csp-lod-bodies#readme). The repo contains two dockerfiles `base.Dockerfile` and `example.Dockerfile`.
+A dockerized mapserver for CosmoScout [csp-lod bodies](https://github.com/cosmoscout/cosmoscout-vr/tree/main/plugins/csp-lod-bodies#readme). This repo has been created taking three use-cases into consideration - `loading example map data`, `loading custom map data` and `creating custom map data containers`. Each of the use-cases are discussed briefly below.
 
-## base.Dockerfile
+## 1) Loading Example Map Data
+Here you run a dockerized mapserver using NASA's `Blue Marble` and the `Natural Earth image` datasets and `ETOPO1` elevation data.
 
-This docker mapserver allows user to use their own dataset. The webserver is exposed on port 80.
-
-```bash
-docker run -ti --rm -p 8080:80 -v "$(pwd)":/mapserver-datasets ghcr.io/cosmoscout/mapserver-base
-```
-
-The command above runs a container, binds `localhost port 8080 to container port 80`, and `mounts the dataset in the pwd to containers /mapserver-datasets `directory.
-
-
-## example.Dockerfile
-
-This docker file uses `base.Dockerfile` as its base image and contains NASA's `Blue Marble` and the `Natural Earth image` datasets and `ETOPO1` elevation data.
+### Running the Server
+To run the dockerized mapserver, you can pull the docker image (built from the `example.Dockerfile`) from GHCR using the command:
 
 ```bash
 docker run -ti --rm -p 8080:80 ghcr.io/cosmoscout/mapserver-example
 ```
+The command binds localhost port 8080 to container port 80.
 
-To ensure that everything is working as intended, see if you can access the following links:
+To ensure that everything is working as intended, see if you can access the example-datasets using the following links:
```

Comment:
I guess that it would be more helpful if the links below were not in a code  block but actual clickable links.